internal/dag/cache.go

```diff
@@ -100,6 +102,27 @@ func (kc *KubernetesCache) matchesIngressClass(obj metav1.Object) bool {
 
 }
 
+// matchesGateway returns true if the given Kubernetes object
+// belongs to the Gateway that this cache is using.
+func (kc *KubernetesCache) matchesGateway(obj metav1.Object) bool {
+
+	fmt.Println("--- obj: ", k8s.NamespacedNameOf(obj))
+	fmt.Println("--- gateway: ", kc.Gateway)
```

Comment:
can drop these now :)

Reply:
bah I'm so bad at leaving those in :headbang: